src/sentry/integrations/discord/utils/channel.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from django.core.exceptions import ValidationError
+
+from sentry.integrations.discord.client import DiscordClient
+from sentry.shared_integrations.exceptions import IntegrationError
+from sentry.shared_integrations.exceptions.base import ApiError
+
+from . import logger
+
+
```

Comment:
New util for checking that we (and the user creating an alert) have access to a server/channel